pkgs/applications/blockchains/turbo-geth.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "turbo-geth";
+  version = "2020.09.04";
+
+  src = fetchFromGitHub {
+    owner = "ledgerwatch";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0dbkpr35aczc2v3rfnahxsnzbhp2kaqnx1i6k1341yssdba13h3k";
+  };
+
+
+  usb = fetchFromGitHub {
+    owner = "karalabe";
+    repo = "usb";
+    rev = "51dc0efba3568b598359930901dc6647e9b2c6a1";
+    sha256 = "0rshv04w5f60zv3l9f0zqyhimpwrbzm7dir8jbkdpnszicq1dx3m";
+  };
+
+  evmc = fetchFromGitHub {
+    owner = "ethereum";
+    repo = "evmc";
+    rev = "v7.3.0";
+    sha256 = "0iw9n9frbzzxi5skb4jngsy4f9gpbri0p3sj0havgprn8bgfh8va";
+  };
+
+  vendorSha256 = "1z07vgkacqplnqxlp971975ilc7mzcash7ygr0rs3y67l4wnayl0";
+
+  overrideModAttrs = (_: {
+    postBuild = ''
+      cp -r --reflink=auto ${usb}/libusb vendor/github.com/karalabe/usb
+      cp -r --reflink=auto ${usb}/hidapi vendor/github.com/karalabe/usb
+      cp -r --reflink=auto ${evmc}/include vendor/github.com/ethereum/evmc/v7
+      cp -r --reflink=auto ${evmc}/lib vendor/github.com/ethereum/evmc/v7
+    '';
+  });
```

Comment:
Just curios, are you aware of `runVend`? https://nixos.org/manual/nixpkgs/unstable/#ex-buildGoModule-2

Reply:
So much nicer! :pray: :sparkles: